src/index.vue

```diff
@@ -10,6 +10,7 @@
         <slot name="default">
           <component :is="spinner" :color="color" :size="`${size}px`" :duration="`${duration}s`"></component>
         </slot>
+        <div :style="{ color }" >{{text}}</div>
```

Comment:
Please use `v-if` to check `text` is existing and please add space inside `{{text}}` like this `{{ text }}`